src/index.tsx

```diff
@@ -0,0 +1,17 @@
+import React from "react"
+import ReactDOM from "react-dom/client"
+import { I18nextProvider } from "react-i18next"
+import { BrowserRouter } from "react-router-dom"
+import i18n from "../i18n"
+import App from "./App"
+import "./styles/global.css"
+
+ReactDOM.createRoot(document.getElementById("root") as HTMLElement).render(
+  <React.StrictMode>
+    <BrowserRouter>
```

Comment:
I'm not convinced about the division: now `BrowserRouter` is split in half between here and `App.tsx` where the `Routes` are

Reply:
In the documentation always appears a single `App` file with `Routes` and an index file with the `BrowserRouter`, also for the translation with i18n put everything in a single file gave me error. However, I couldn’t find any indication of the best practices to follow so I assumed this was the correct one

Reply:
Do you have a link for the docs? I wasn't able to find it

Reply:
Here you can directly find the code from the examples on [github](https://github.com/remix-run/react-router/tree/main/examples)

Reply:
Thanks! I was looking at `i18next` docs and not finding anything...😅 

The notation confuses me a bit, but they indeed use it everywhere so I guess there must be a reason